tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/call_settings.py

```diff
@@ -0,0 +1,211 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import Base
+
+
+class CallSettings(Base):
+
+    _page_locator = (By.ID, 'call-iccs')
+    _sim_1_locator = (By.ID, 'menuItem-call-sim1')
+
+    _cs_page_locator = (By.ID, 'call')
+    _voicemail_locator = (By.CSS_SELECTOR, '.menuItem-voicemail')
+    _voicemail_number_locator = (By.CLASS_NAME, 'vm-number')
+    _voicemail_page_locator = (By.ID, 'call-voiceMailSettings')
+
+    _caller_id_menu_item_locator = (By.ID, 'menuItem-callerId')
+    _caller_id_selector_locator = (By.ID, 'ril-callerId')
+    _caller_id_confirm_button_locator = (By.CLASS_NAME, 'value-option-confirm')
+
+    _fd_page_locator = (By.ID, 'call-fdnSettings')
+    _fixed_dialing_numbers_locator = (By.ID, 'menuItem-callFdn')
+    _toggle_fixed_dialing_number_locator = (By.CSS_SELECTOR, '.fdn-enabled .pack-switch')
+    _sim_pin_page_locator = (By.ID, 'simpin-dialog')
+    _sim_pin_area_locator = (By.CLASS_NAME, 'sim-code-area')
+    _auth_number_locator = (By.CSS_SELECTOR, '[data-l10n-id = "fdn-authorizedNumbers"]')
+    _auth_number_page_locator = (By.ID, 'call-fdnList')
+
+    _cf_page_locator = (By.ID, 'call-forwarding')
+    _call_forwarding_locator = (By.ID, 'menuItem-callForwarding')
+    _cf_status_disabled_text_locator = (By.CSS_SELECTOR, '[data-l10n-id = "callForwardingNotForwarding"]')
+    _call_forwarding_always_locator = (By.ID, 'li-cfu-desc')
+    _cf_always_page_locator = (By.ID, 'call-cf-unconditional-settings')
+    _call_forwarding_busy_locator = (By.ID, 'li-cfmb-desc')
+    _cf_busy_page_locator = (By.ID, 'call-cf-mobile-busy-settings')
+    _call_forwarding_unanswered_locator = (By.ID, 'li-cfnrep-desc')
+    _cf_unanswered_locator = (By.ID, 'call-cf-no-reply-settings')
+    _call_forwarding_unreachable_locator = (By.ID, 'li-cfnrea-desc')
+    _cf_unreachable_locator = (By.ID, 'call-cf-not-reachable-settings')
+
+    _cb_page_locator = (By.ID, 'call-cbSettings')
```

Comment:
This is Call Barring. Perhaps it would be a good idea to give a comment indicating what these abbreviations stand for? What Settings page they represent.